src/components/measureList/MeasureList.tsx

```diff
@@ -91,15 +94,28 @@ export default function MeasureList(props: { measureList: Measure[] }) {
                         {measure.revisionNumber}
                       </td>
                       <td tw="px-6 py-4 whitespace-nowrap text-right text-sm font-medium">
-                        <button
-                          onClick={() => {
-                            history.push(`/measures/${measure.id}/edit`);
-                          }}
-                          tw="text-blue-600 hover:text-blue-900"
-                          data-testid={`edit-measure-${measure.id}`}
-                        >
-                          Edit
-                        </button>
+                        {measure.createdBy === userName && (
+                          <button
+                            onClick={() => {
+                              history.push(`/measures/${measure.id}/edit`);
+                            }}
+                            tw="text-blue-600 hover:text-blue-900"
+                            data-testid={`edit-measure-${measure.id}`}
+                          >
+                            Edit
+                          </button>
+                        )}
+                        {measure.createdBy !== userName && (
+                          <button
+                            onClick={() => {
+                              history.push(`/measures/${measure.id}/edit`);
+                            }}
+                            tw="text-blue-600 hover:text-blue-900"
+                            data-testid={`view-measure-${measure.id}`}
```

Comment:
do we need these to be two different buttons or could they be just one button with the button text driven by if the user is the createdBy user?

Reply:
@nmorasb I made different test cases to get the different test ids, that was why.

Reply:
that could also be addressed by fetching the button by role + name instead of by testId in the tests

Reply:
@sb-cecilialiu I think the `data-testid`s can be dynamic, too
```suggestion
                            data-testid={`{action}-measure-${measure.id}`}
```

Reply:
@nmorasb I removed one button.